src/transform/HeaderTransform.css

```diff
@@ -0,0 +1,63 @@
+.pagelib_edit_section_link {
```

Comment:
This would need to be scoped to just the edit pencil used by this transform. Easiest way could be placing the edit pencil in a div which this transform floats as needed.

Reply:
Done